engine/src/parser/expression_utils.cpp

```diff
@@ -285,6 +287,7 @@ operator_type map_to_operator_type(const std::string & operator_token) {
 		{"CHAR_LENGTH", operator_type::BLZ_CHAR_LENGTH},
 		{"LOWER", operator_type::BLZ_STR_LOWER},
 		{"UPPER", operator_type::BLZ_STR_UPPER},
+                {"INITCAP", operator_type::BLZ_STR_INITCAP},
```

Comment:
please  fix spacing so that it matches the rest

Reply:
addressed